src/components/ContentUploader/ContentUploader.js

```diff
@@ -561,6 +563,9 @@ class ContentUploader extends Component<Props, State> {
                 totalNumOfItems >= EXPAND_UPLOADS_MANAGER_ITEMS_NUM_THRESHOLD &&
                 useUploadsManager
             ) {
+                this.setState({
```

Comment:
I can add a line to the above if statement checking:
!isUploadsManagerExpanded
So it only sets autoexpanded if not manually expanded.

I can also set isAutoExpanded to true when an error happens since it autoexpands in this case too

Reply:
I would say add this check. It's minor but technically a bug